docs/docs-content/clusters/cluster-management/cluster-proxy.md

```diff
@@ -0,0 +1,223 @@
+cluster to use the proxy server. You can do this by applying the `spectrocloud.com/connection: proxy` label to the
+specific job, deployment, or daemon set to instruct an application to use the proxy settings of the cluster.
+
+## Prerequisites
+
+- An active proxy server reachable by your cluster.
+
+<Tabs groupId="scenario">
+
+<TabItem value="Palette SaaS Non-Edge">
+
+- A PCG is deployed into an active and healthy Kubernetes cluster. Refer to
+  [Deploy a PCG to an Existing Kubernetes Cluster](../pcg/deploy-pcg-k8s.md) for additional guidance.
+
+  :::warning
+
+  If you deployed a [PCG through the Palette CLI](../pcg/pcg.md#supported-environments), refer to the respective
+  platform installation guide for instructions on how to configure proxy settings during the installation process
+  through the CLI.
+
+  :::
+
+- The PCG is configured to use the proxy server that you intend for your applications to use for outbound
+  communications. For more information, refer to
+  [Enabled and Manage Proxy Configurations](../pcg/manage-pcg/configure-proxy.md).
+
+</TabItem>
+
+<TabItem value="Self-Hosted Palette Non-Edge">
+
+- A self-hosted Palette instance is deployed into an active and healthy Kubernetes cluster. Refer to
+  [Self-Hosted Palette Installation](../../enterprise-version/install-palette/install-palette.md) for additional
+  guidance.
+
+- The self-hosted Palette instance is configured to use the proxy server that you intend for your applications to use
+  for outbound communications.
+
+</TabItem>
+
+<TabItem value="Edge">
+
+- Your Edge host is configured to use the proxy server that you intend for your applications to use for outbound
+  communications. Refer to [Edge Host Installation](../edge/site-deployment/stage.md) for more information.
+
+</TabItem>
+
+</Tabs>
+
+## Configure Applications to Use Proxy Server
+
+<Tabs groupId="scenario">
+
+<TabItem value="Palette SaaS Non-Edge">
+
+1.  If you are using Palette SaaS, and your cluster does not have direct access to the internet, then the communication
+    between Palette SaaS and your cluster must go through a Private Cloud Gateway (PCG). For more information about
+    PCGs, refer to [Private Cloud Gateway](../pcg/pcg.md). You must configure proxy settings for the PCG, and then
+    clusters created by cloud accounts configured to use the PCG will inherit its proxy settings.
+
+    If you are provisioning the PCG using the Palette CLI, you can configure the proxy settings during the PCG
+    installation through the Palette CLI's interactive prompts. If you are using Helm to provision a PCG or have an
+    existing PCG that is not yet configured to use proxy, refer to
+    [Enable and Manage Proxy Configurations for PCG](../pcg/manage-pcg/configure-proxy.md) to learn how to install Reach
+    on a PCG cluster and use it to configure proxy settings.
+
+2.  Once you have configured proxy settings through Reach for your PCG, you can proceed to create your cluster using a
```

Comment:
SaaS customers will have to register a new cloud account in Palette and specify the PCG. They cannot alter existing cloud accounts. I think it would help if we link to the three public cloud accounts registration guides to help the user.